src/internal/config.ts

```diff
@@ -17,5 +19,17 @@ export const config = {
    * an unhandled error. DO NOT USE THIS FLAG UNLESS IT'S NEEDED TO BY TIME
    * FOR MIGRATION REASONS.
    */
-  useDeprecatedSynchronousErrorHandling: false,
+  set useDeprecatedSynchronousErrorHandling(value: boolean) {
+    if (value) {
+      const error = new Error();
+      console.warn('DEPRECATED! RxJS was set to use deprecated synchronous error handling behavior by code at: \n' + error.stack);
+    } else if (_enable_super_gross_mode_that_will_cause_bad_things) {
+      console.log('RxJS: Back to a better error behavior. Thank you. <3');
```

Comment:
I don't think this log message is necessary. let's leave warn only.

Reply:
I added this in part so we can see that it's removed in the tests (which turn it on then off in the beforeEach and afterEach)

Reply:
Since test itself does verify behavior of config, is it really needed to observe console logs for it?